src/app/component/profile/profile.json

```diff
@@ -0,0 +1,214 @@
+{
```

Comment:
is this json schema?
is it valid?

it should probable have the `$schema` key in it, not least for utilization of vscode intellisense http://json-schema.org/schema#

Reply:
https://github.com/QutEcoacoustics/workbench-client/issues/181